crates/cmds-solana/src/nft/create_metadata_account.rs

```diff
@@ -200,8 +132,8 @@ pub struct InputStruct {
     pub metadata: NftMetadata,
     pub metadata_uri: String,
     pub uses: NftUses,
-    // #[serde(with = "value::pubkey::opt")]
-    // pub collection_mint_account: Option<Pubkey>,
```

Comment:
Did you encounter any problem with this previously?

Reply:
The opt; can't get it to work

Reply:
Fixed